server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/PullrequestRepository.java

```diff
@@ -0,0 +1,15 @@
+package de.tum.in.www1.hephaestus.codereview.pullrequest;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface PullRequestRepository extends JpaRepository<PullRequest, Long> {
+
+    @Query("SELECT p FROM PullRequest p, User a WHERE p.author = a AND a.login = ?1")
+    List<PullRequest> findByAuthor(String authorLogin);
```

Comment:
You can write it like this and use a `Set`:
```suggestion
    Set<PullRequest> findByAuthor_Login(String authorLogin);
```

Reply:
That's very interesting! Is there maybe a good "documentation" for the naming I could learn from?

Reply:
https://docs.spring.io/spring-data/jpa/reference/repositories/query-methods-details.html

i will also read through it, maybe we don't even need the `_`